package.json

```diff
@@ -2,7 +2,7 @@
   "name": "luminous-lightbox",
   "version": "2.0.0",
   "description": "A simple, lightweight, no-dependencies JavaScript image lightbox.",
-  "main": "dist/Luminous.min.js",
+  "main": "lib/lum-cjs.js",
```

Comment:
Is `-cjs.js` a standard convention? If it's not, I'd rather see `-common` or `-commonjs`, just for clarity.

Reply:
It's not enough of a convention to choose it over `-commonjs`. Updating now